### PR TITLE
Fix interoperability with NTPSEC and make compliant to latest draft

### PR DESCRIPTION
**Pull request by mdavids**
As discussed with Watson by e-mail.

### CODE REVIEW

File: src/nts_ke/client.rs [hunk 1]
```diff
@@ -269,7 +270,7 @@ pub fn run_nts_ke_client(
         }
     }
     debug!(logger, "saw the end of the response");
-    stream.shutdown(Shutdown::Both)?;
+    stream.shutdown(Shutdown::Write)?;
```
Comment (haxxpop) [inline src/nts_ke/client.rs]: Why not both?

Comment (haxxpop) [inline src/nts_ke/client.rs]: Why not both?

Comment (mdavids) [reply src/nts_ke/client.rs]: Because that doesn't seem to work on NTPSEC.

Try: cfnts client ntp1.glypnod.com -p 123

Comment (wbl) [reply src/nts_ke/client.rs]: What's happening is the close notify comes in after we shut down the reading half, hence triggering a failure.

Comment (haxxpop) [reply src/nts_ke/client.rs]: I see

**Comment by haxxpop** [comment]
Thank you for your contribution :)

**Pull request merged**